<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetQuestion</class>
 <widget class="QMainWindow" name="SetQuestion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SetQuestion</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="hlBox">
      <item>
       <widget class="QRadioButton" name="rbDir1">
        <property name="text">
         <string>easy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbDir2">
        <property name="text">
         <string>middle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbDir3">
        <property name="text">
         <string>hard</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbDir4">
        <property name="text">
         <string>very hard</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>input questions</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="ieNumber1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="ieOper"/>
       </item>
       <item>
        <widget class="QLineEdit" name="ieNumber2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="ieIsEqual"/>
       </item>
       <item>
        <widget class="QLineEdit" name="ieResult"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lbRight">
         <property name="text">
          <string>right</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ieRight">
         <property name="styleSheet">
          <string notr="true">color:rgb(0, 255, 0);
text-align:center;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbError">
         <property name="text">
          <string>error</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ieError">
         <property name="styleSheet">
          <string notr="true">color:rgb(255, 0, 0);
text-align:center;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbTotal">
         <property name="text">
          <string>total</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ieTotal">
         <property name="styleSheet">
          <string notr="true">text-align:cneter;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>time</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ieTime"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pbCheck">
        <property name="text">
         <string>check</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbStatics">
        <property name="text">
         <string>statics</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbExit">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>466</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
